crates/project/src/terminals.rs

```diff
@@ -128,6 +141,41 @@ impl Project {
         None
     }
 
+    pub fn set_python_venv_path_for_tasks(
+        &mut self,
+        settings: &VenvSettingsContent,
+        working_directory: Option<PathBuf>,
+        env: &mut collections::HashMap<String, String>,
+    ) {
+        let working_directory = working_directory.unwrap_or_else(|| Path::new("/").to_path_buf());
```

Comment:
Let's at least deal with the comment from
https://github.com/zed-industries/zed/blob/5139aa38117a36e9e55abdcb047b9eafc84c6f45/crates/project/src/terminals.rs#L107-L109

I also wonder, whether defaulting to `/`+arbitrary string from the config is a good default.
For alacritty, `working_directory` boils down to `Command::process` behavior of Rust stdlib:
https://github.com/alacritty/alacritty/blob/406170050d1b08a0ad717d9e6746c6369bbf262a/alacritty_terminal/src/tty/unix.rs#L251-L254

which defaults to relative path usage.
Could it also be more appropriate for python env case?

Reply:
That was not my decision to make, so I'd say that is intended behaviour.

Reply:
Sure, this works for me